lib/Runtime/Library/UriHelper.cpp

```diff
@@ -126,10 +126,13 @@ namespace Js
     }
 
     // The Encode algorithm described in sec. 15.1.3 of the spec. The input string is
-    // 'input' and the Unescaped set is described by the flags 'unescapedFlags'. The
+    // 'strURI' and the Unescaped set is described by the flags 'unescapedFlags'. The
     // output is a string var.
-    Var UriHelper::Encode(__in_ecount(len) const  char16* input, uint32 len, unsigned char unescapedFlags, ScriptContext* scriptContext )
+    Var UriHelper::Encode(JavascriptString* strURI, unsigned char unescapedFlags, ScriptContext* scriptContext )
     {
+        uint32 len = strURI->GetLength();
```

Comment:
I realise that this is how it was previously, but should this be a `charcount_t`?

Reply:
Yep

---
In reply to: [205613734](https://github.com/Microsoft/ChakraCore/pull/5540#discussion_r205613734) [](ancestors = 205613734)